staging/onex_model/setup.py

```diff
@@ -4,7 +4,7 @@
 import setuptools
 
 pkg = "onex_model"
-version = "0.0.13"
+version = "0.0.14"
```

Comment:
mainline is already on 0.0.14 (for all 3 onex packages). To publish a new version after merge you'll need to bump it to 0.0.15

(and this information should really be derived from git tags)